src/select/typeahead/header/TypeaheadSelectHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import "./_typeahead-select-header.scss";
+
+import React, {Fragment} from "react";
+import classNames from "classnames";
+
+import ListItem from "../../../list/item/ListItem";
+import Tag, {TagShape} from "../../../tag/Tag";
+
+export interface TypeaheadSelectHeaderProps {
+  tags: TagShape[];
+  handleTagRemove: (tag: TagShape) => void;
+  customClassName?: string;
+  input?: React.ReactNode;
+}
+
+function TypeaheadSelectHeader({
+  tags,
+  customClassName,
+  handleTagRemove,
+  input
+}: TypeaheadSelectHeaderProps) {
+  const listClassName = classNames("typeahead-select__header-list", customClassName);
+
+  return (
+    <ul className={listClassName}>
```

Comment:
```suggestion
    <ul className={classNames("typeahead-select-header__tag-list", customClassName);}>
```
We can omit the variable to make the file more clean and simple